### PR TITLE
add filter to prepare builder data before display

### PR DESCRIPTION
**Pull request by andreiglingeanu**

### CODE REVIEW

File: class-fw-extension-page-builder.php [hunk 1]
```diff
@@ -284,6 +284,8 @@ private function get_post_content_shortcodes(WP_Post $post) {
 			&&
 			$builder_data['builder_active']
 		) {
+			$builder_data = apply_filters('fw_prepare_builder_data_before_display', $builder_data);
```
Comment (moldcraft) [inline class-fw-extension-page-builder.php]: Please rename to `fw:ext:page-builder:builder-data:before-shortcode-generate`

I started to use this convention `fw:ext:{extension}:{something}`

**Comment by andreiglingeanu** [comment]
@moldcraft is there something else that I can do in order to get this pull request merged?

**Comment by moldcraft** [comment]
Sorry, I forget to attach it to me and forget about it (I check all issues on [this page](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+assignee%3Amoldcraft))

**Comment by andreiglingeanu** [comment]
It's okay. Thank you.

**Pull request merged**